topi/python/topi/cuda/conv2d.py

```diff
@@ -105,7 +105,9 @@ def conv2d_cuda(cfg, data, kernel, strides, padding, dilation, layout='NCHW', ou
         return winograd_cuda(cfg, data, kernel, strides, padding, dilation, layout, out_dtype,
                              pre_computed=False)
     if cfg.template_key == 'int8':
-        return conv2d_NCHWc_int8(cfg, data, kernel, strides, padding, dilation, layout, out_dtype)
+        if (data.dtype == 'int8' or data.dtype == 'uint8'):
```

Comment:
Does it work with `uint8`? [conv2d compute](https://github.com/dmlc/tvm/blob/fa38ef078b7d7e67920e9bd7be0d391b3412518e/topi/python/topi/cuda/conv2d_int8.py#L134) and [dp4a](https://github.com/dmlc/tvm/blob/fa38ef078b7d7e67920e9bd7be0d391b3412518e/topi/python/topi/cuda/conv2d_int8.py#L152) need some changes since they are hardcoded as int8 (although it should work with uint8)

Reply:
You're right that it only supports int8 now but could be extended to uint8. I added this check because I was doing some quantization without autotuning, which caused the incorrect convolution algorithm to be chosen in some cases. Since autotuning is the intended workflow I think dropping this would be fine.